frontend/src/pages/PipelineList.tsx

```diff
@@ -176,6 +167,8 @@ class PipelineList extends Page<{}, PipelineListState> {
         request.pageSize,
         request.sortBy,
         request.filter,
+        this.props.namespace ? 'NAMESPACE' : undefined,
```

Comment:
I am not sure if the backend has already implemented the logics to support namespace-based listPipelines API call. cc @chensun to confirm.

Reply:
Namespaced pipelines are supported since https://github.com/kubeflow/pipelines/pull/4835 PR.